modules/ROOT/pages/configuring-mule-stacktraces.adoc

```diff
@@ -1,69 +1,71 @@
-= Configuring Mule Stacktraces
+= Configuring Mule Stack Traces
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: anypoint, studio, mule, on premises, on premise, amc, logs, stacktraces, verbose
+:keywords: anypoint, studio, mule, on premises, on premise, amc, logs, stack traces, verbose
 
-By default Mule does not show stacktraces and even filters out some internal class
- references from stacktraces to  produce a more readable output when it does show them.
-This behavior can be controlled in one of two ways:
+By default, Mule does not display all stack traces. When Mule does show stack traces, it filters out some internal class references from to produce a more readable output.
 
-* Command-line arguments
+You can control Mule stacktrace behavior in two ways:
 
-* Setting properties in Mule Studio
+* Use command-line arguments
 
-== Command-Line Arguments
+* Set properties in Mule Studio
 
-Two command-line properties let you control this behavior using `-M-DpropertyName`:
+== Use Command-Line Arguments
 
-* `mule.verbose.exceptions` - If set to *true*, Mule shows filtered stacktraces.
- Intended for Mule developers and administrators only and while troubleshooting,
- end users probably never need to enable this.
+Use `-M-DpropertyName` with the following command-line arguments to control the behavior of stacktraces:
+
+* `mule.verbose.exceptions` - If set to *true*, Mule shows filtered stack traces.
 
 * `mule.stacktrace.filter` - A comma-separated list of packages and/or classes to
- remove from the stacktraces, which are matched via `string.startsWith()`.
+ remove from the stack traces, which are matched using `string.startsWith()`.
+
+ Note: These arguments are available for Mule developers and administrators to use while troubleshooting. They are not intended for use by end users.
 
-== Setting Properties in Anypoint Studio
+== Set Properties in Anypoint Studio
 
-In Studio, you can set the same properties specified above to instruct Mule to
-provide either stacktraces or to configure a particular filter of
-packages or classes to remove. To adjust these settings, open the `Run Configurations...`
+In Studio, you can set the properties specified above to instruct Mule to
+provide stack traces or to filter out packages or classes to remove. To adjust these settings, open the `Run Configurations...`
  of your project and define the properties to specify the behavior you want. In the
  example below, the `mule.verbose.exceptions` property is set to *true*.
 
 image::stack-traces-studio.png[Studio-config-stacktrace]
 
-== Obtaining the Stacktrace and Thread Dump
+== Obtain the Stack Trace and Thread Dump
+
+Follow these steps to troubleshoot Mule if stops and you need
```

Comment:
`if it stops`?

Reply:
This was in the original doc and I agree that it is unclear. Are one of these lines this better?
...if Mule stops running
...if the Mule Runtime engine stops running

Reply:
I think the first is better